lib/testPaymentExpiry.js

```diff
@@ -0,0 +1,191 @@
+'use strict'
+const _ = require('lodash')
+const moment = require('moment')
+const request = require('co-request')
+const UnacceptableExpiryError = require('../errors/unacceptable-expiry-error')
+
+module.exports = function * (config, payment) {
+  let tester = new TransferTester(config, payment)
+  yield tester.loadCaseExpiries()
+  return tester
+}
+
+function TransferTester (config, payment) {
+  this.config = config
+  this.source_transfers = payment.source_transfers
```

Comment:
It looks like most of the code is using camel case, and that is javascript's built-in case, so I think we should probably stick with that everywhere possible i.e. everywhere that it is not going to a JSON object.